docs/zkapps/tutorials/06-offchain-storage.mdx

```diff
@@ -84,10 +84,10 @@ Suggested improvements:
 
 ## Implement a Project Using Off-Chain Storage
 
-The sample code for this project is provided at [examples/zkapps/06-offchain-storage/contracts](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/06-offchain-storage/contracts) with a focus on:
+The sample code for this project is provided at [examples/zkapps/06-offchain-storage/contracts](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/06-offchain-storage/offchain-storage-zkapp/contracts) with a focus on:
 
-- [src/main.ts](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/06-offchain-storage/contracts/src/main.ts)
-- [src/NumberTreeContract.ts](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/06-offchain-storage/contracts/src/NumberTreeContract.ts) 
+- [src/main.ts](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/06-offchain-storage/offchain-storage-zkapp/contracts/src/main.ts)
```

Comment:
thank you! TIL when to use blob and file (did a little web searching to verify, you are spot on)
I have updated the style guide with this guidance
https://github.com/o1-labs/docs2/wiki/Markdown-Syntax#links